src/Components/WebAssembly/Server/src/DebugProxyLauncher.cs

```diff
@@ -111,16 +116,17 @@ private static void PassThroughConsoleOutput(Process process)
 
         private static void CompleteTaskWhenServerIsReady(Process aspNetProcess, TaskCompletionSource<string> taskCompletionSource)
         {
-            string capturedUrl = null;
+            string? capturedUrl = null;
             aspNetProcess.OutputDataReceived += OnOutputDataReceived;
             aspNetProcess.BeginOutputReadLine();
 
             void OnOutputDataReceived(object sender, DataReceivedEventArgs eventArgs)
             {
-                if (String.IsNullOrEmpty(eventArgs.Data))
+                if (string.IsNullOrEmpty(eventArgs.Data))
                 {
                     taskCompletionSource.TrySetException(new InvalidOperationException(
-                            "No output has been recevied from the application."));
+                        "No output has been recevied from the application."));
+                    return;
```

Comment:
@captainsafia this looks like a bug.  The current code would continue to execute and result in a null-ref in the next if

Reply:
Yes, good catch. Thanks!